### PR TITLE
Add container

### PR DESCRIPTION
**Pull request by melyxlin**
Since the Dockerfile was giving the backend some error, I switched to using separate docker files for the frontend and backend and a docker-compose file that will run those two files simultaneously. This both a standalone deployment for the frontend and backend as well as a complete deployment of the web app

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -25,22 +25,34 @@ $ cd oauth2l-web/web
 ### Docker
 Download [Docker](https://docs.docker.com/get-docker/) if not already installed. 
 
-Create the image by running the following command:
+#### Running the Web Application
+Create the images for the backend and the frontend by running the following command:
 ```bash
-$ docker build -t oauth2l-web .
+$ docker-compose up -d --build .
 ```
```
Comment (LeoRuiz01) [inline web/README.md:32]: Can this be run anywhere from oauth2l-web or do you have to be in the web directory?

File: web/README.md [hunk 2]
```diff
@@ -25,22 +25,34 @@ $ cd oauth2l-web/web
...
+```
+Then run the image with the following command:
+```bash
+$ docker run --detach --name oauth2l-web-frontend 3000:3000 -d oauth2l-web #frontend
+$ docker run --detach --name oauth2l-web-backend 8080:8080 -d oauth2l-web #backend
+```
+To stop the image, run the following command
+```bash
+$ docker container stop oauth2l-web-<frontend/backend>
+$ docker container rm oauth2l-web-<frontend/backend>
```
Comment (LeoRuiz01) [inline web/README.md:54]: The reason you need to run these two commands is because of there's two docker images right?

File: web/README.md [hunk 3]
```diff
@@ -25,22 +25,34 @@ $ cd oauth2l-web/web
 ### Docker
 Download [Docker](https://docs.docker.com/get-docker/) if not already installed.
```
Comment (LeoRuiz01) [inline web/README.md:26]: Shouldn't this also include Docker Compose to be installed?

File: web/README.md (hunk 2 above)
Comment (melyxlin) [reply web/README.md:54]: yep.

File: web/README.md (hunk 3 above)
Comment (melyxlin) [reply web/README.md:26]: It is already preinstalled with docker.

**Pull request merged**